src/python/grpcio/tests/unit/framework/interfaces/links/test_utilities.py

```diff
@@ -51,7 +51,7 @@ def _safe_for_log_ticket(ticket):
       possibly features values like the string "<payload of length 972321>" in
       place of the actual values of the given ticket.
   """
-  if isinstance(ticket.payload, (basestring,)):
+  if isinstance(ticket.payload, (str,)):
```

Comment:
We've already decided to use `six`, and as best I can tell the `builtins` module from `future` would require breakage from our current style guide to provide ergonomic gains. That said...

This is an interesting suggestion as far as choice-of-dependencies go. Not immediately desirable or even desirable in the long run (I'd rather cythonize in the long run), but mid-term it *might* be worth considering.